<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="backgroundwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{ 
background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));}</string>
   </property>
   <widget class="QLabel" name="backgroundPhoto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="byPictureButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 14pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>BY
PICTURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FACButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 22pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>Football Action
Classification</string>
    </property>
   </widget>
   <widget class="QLabel" name="howItWorksText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)</string>
    </property>
    <property name="text">
     <string>Some stuff to know before analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="howItWorksHeadText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190);
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Start Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seeVisualsButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 13pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>SEE
VISUALS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAnalysisButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 13pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>START
ANALYSIS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;

	background-color: rgb(255, 255, 255);

	font: 75 14pt; color: rgb(0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
